addon/components/cfb-form-editor/question-list/item.hbs

```diff
@@ -50,11 +50,33 @@
       <span class="uk-width-auto uk-margin-small-right uk-text-bold">{{@question.slug}}</span>
     {{/if}}
 
-    <span
-      class="uk-width-expand uk-margin-small-right uk-text-small uk-text-muted uk-text-truncate"
+{{#let
+  (and 
+    (eq @mode "reorder")
+    (or
+      (has-question-type @question "form")
+      (has-question-type @question "table")
+    )
+  )
+  as |showFormLink|
+}}
```

Comment:
Let's put this in the JS file. This makes the template hard to read..

Reply:
Fixed